hackathon_site/hackathon_site/settings/__init__.py

```diff
@@ -272,10 +272,10 @@
 HACKATHON_NAME = "MakeUofT"
 CONTACT_EMAIL = DEFAULT_FROM_EMAIL
 
-REGISTRATION_OPEN_DATE = datetime(2020, 9, 1, tzinfo=TZ_INFO)
-REGISTRATION_CLOSE_DATE = datetime(2023, 9, 30, tzinfo=TZ_INFO)
-EVENT_START_DATE = datetime(2023, 10, 10, 10, 0, 0, tzinfo=TZ_INFO)
-EVENT_END_DATE = datetime(2023, 10, 11, 17, 0, 0, tzinfo=TZ_INFO)
+REGISTRATION_OPEN_DATE = datetime(2022, 1, 3, tzinfo=TZ_INFO)
+REGISTRATION_CLOSE_DATE = datetime(2022, 2, 5, tzinfo=TZ_INFO)
+EVENT_START_DATE = datetime(2022, 2, 19, 12, 0, 0, tzinfo=TZ_INFO)
+EVENT_END_DATE = datetime(2022, 2, 20, 12, 0, 0, tzinfo=TZ_INFO)
```

Comment:
do the `0, 0` indicate hour and time of the day? If so, wouldn't it be something like 9am start and 5pm end

Reply:
oh wait I see above its like 12PM -12PM, in that case should it be `12, 0` @grahamhoyes

Reply:
it's 12, 0 rn the last 0 is seconds i'm pretty sure